app/controllers/support/providers/accredited_providers_controller.rb

```diff
@@ -49,8 +49,30 @@ def update
         end
       end
 
+      def delete
+        @cannot_delete = provider.courses.exists?(accredited_provider_code: accredited_provider.provider_code)
```

Comment:
I would just memoize this variable as a method then you could do `cannot_delete?`